core/src/main/java/com/linecorp/armeria/server/ServiceRequestContextBuilder.java

```diff
@@ -169,6 +158,8 @@ public ServiceRequestContext build() {
         final RoutingResult routingResult =
                 this.routingResult != null ? this.routingResult
                                            : RoutingResult.builder().path(path()).query(query()).build();
+        final InetAddress clientAddress = server.config().clientAddressMapper().apply(proxiedAddresses)
+                                                .getAddress();
```

Comment:
Seems like `clientAddressMapper` is not configurable when using `ServiceRequestContext`. Is this intentional?

Reply:
I thought `clientAddressMapper` could be configurable with `serverConfigurator(Consumer<? super ServerBuilder> serverConfigurator)`.